src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -105,96 +106,6 @@ class QueryAppsDataValidator {
     return true;
   }
 
-  bool ValidateAppDataAndOsAndLanguagesData() const {
```

Comment:
@anosach-luxoft , good catch. `ValidateLanguages`, `ValidateAppIdAndAppName` never used now. I'll delete it too.